storage/scylla_table.h

```diff
@@ -81,6 +81,39 @@ status_t db_set_identity_id(db_identity_t* obj, cass_int64_t id);
  */
 cass_int64_t db_ret_identity_id(const db_identity_t* obj);
 
+/**
+ * @brief set time(seconds) in db_identity_t
+ *
+ * @param[in] obj pointer to db_identity_t
+ * @param[in] time time to be set into db_identity_t
+ * @return
+ * - SC_OK on success
+ * - SC_TA_NULL/SC_STORAGE_INVAILD_INPUT on error
+ */
+status_t db_set_identity_time(db_identity_t* obj, cass_int64_t time);
+
+/**
+ * @brief return time(seconds) in db_identity_t
+ *
+ * @param[in] obj pointer to db_identity_t
+ *
+ * @return
+ * - time on success
+ * - NULL on error
+ */
+cass_int64_t db_ret_identity_time(const db_identity_t* obj);
```

Comment:
Why did you name it `db_ret_identity_time` rather than `db_get_identity_time`?

Reply:
Name `ret_xxx()` when functions will return the results.
Name `get_xxx(arg)` when functions save results into arguments.

Reply:
You shall clarify the conventions and unify the whole source tree as well.

Reply:
Could I leave it unchanged here and open an issue for it?
Or revise it as `db_get_identity_time` first.